src/index.html

```diff
@@ -8,6 +8,76 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0">
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
```

Comment:
Structure is not overloaded with text, so we don't have to add empty line between sibling elements here.